### DISCUSSION TITLE
Αξιολόγηση για Ομαδικότητα και Συνεργασία

### DISCUSSION THREAD

**Discussion by Sola1r3** [courses-ionio/hci#1643] [Announcements]
Όσον αφορά την συμμετοχή στις συζητήσεις εδώ στο github, από ποιο σημείο και μετά μπορούμε να πούμε οτι μιλάμε για μια αξιόλογη συμμετοχή; Για παράδειγμα 20-30 replys είναι αρκετά για να πούμε ότι βαθμολογείται η συμμετοχή με 7; 

Επιπλέον, τα στάνταρ της αξιολόγησης σε αυτό το κομμάτι θα είναι και συγκριτικά; Θα συγκριθεί δηλαδή η δική μου συμμετοχή όπου εγω ενδεχομένως να είχα 30 ας πούμε ποιοτηκά replys στο τέλος του εξαμήνου αλλά ο άλλος που έχει συγκεντρώσει 60 ανεβάζει τα στάνταρ και βαθμολογείται παραπάνω;

**Comment by marioskomianos** [comment]
δεν νομιζω να ανεβαζει τα στανταρ απο την στιγμη που δεν εχουν μαζευτει αρκετα replies..

**Comment by p14kairas** [comment]
Eπίσης δεν νομίζω ουτε ούτε τα replies να παίζουν ρόλο. Όταν απαντάει κάποιος άσχετα με τα replies το κάνει για να βοηθήσει.

**Reply by Sola1r3** [reply]
Σίγουρα είναι και από καλή θέληση, αλλά τα replies παίζουν ρόλο σε αυτό το κομμάτι του βαθμού. Η συνολική εικόνα μετράει στα φορουμ και δεν είναι απαραίτητο να αποσπάσεις helpful για να βαθμολογηθείς με καλό βαθμό, όσο το να είσαι γενικά ενεργός και παράλληλα να το κάνεις με ουσιαστικό τρόπο.

**Comment by epidrome** [comment]
η ποιότητα είναι πιο σημαντική από την ποσότητα!

σίγουρα θα παίξει ρόλο και το συγκριτικό

εκτός από τις απαντήσεις μετράνε και οι ερωτήσεις

επίσης κάποιος μπορεί να χάσει βαθμούς για άσκοπες ερωτήσεις-απαντήσεις, π.χ., αν υπάρχουν ήδη εδώ ή στο υλικό του μαθήματος

περισσότερο όμως μετράει η αξιολόγηση που θα κάνετε στα τελικά αιτήματα ενσωμάτωσης, όπως λένε οι οδηγίες:

> Για την βαθμολογία με την βάση (5, πέντε) αρκεί να υπάρχει εποικοδομητική συμμετοχή στις συζητήσεις (π.χ., χρήσιμες ερωτήσεις, σωστές απαντήσεις), για πολύ καλή βαθμολογία απαιτείται σημαντική, έγκαιρη, και συχνή συμμετοχή, ενώ για την άριστη βαθμολόγηση θα πρέπει να γίνει [αξιολόγηση των αιτημάτων ενσωμάτωσης της τελικής αναφοράς](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).

για να γίνουμε περισσότερο συγκεκριμένοι, δυστυχώς, με τα μέχρι τώρα δεδομένα, εκτός από κάποιους λίγους, οι περισσότεροι θα έχουν πρόβλημα με αυτό το κριτήριο για αυτό άλλωστε βγάλαμε έγκαιρα σχετική ανακοίνωση #1629 με την ελπίδα ότι θα αλλάξουν συμπεριφορές, αφού είναι σίγουρο ότι παρόμοιες συζητήσεις γίνονται σε άλλα κανάλια
